ArcGISRuntimeSDKQt_CppSamples/Maps/MapReferenceScale/MapReferenceScale.cpp

```diff
@@ -48,11 +49,6 @@ MapReferenceScale::MapReferenceScale(QObject* parent /* = nullptr */):
     m_layerInfoListModel = m_map->operationalLayers();
     emit layerInfoListModelChanged();
     emit currentMapScaleChanged();
-
-    connect(m_mapView, &MapQuickView::mapScaleChanged, this, [this]()
-    {
-      emit currentMapScaleChanged();
-    });
```

Comment:
This should stay. `mapScaleChanged` is not deprecated, and can emit as you zoom the map. It is not solely the result signal for `setViewpointScaleAsync`.